rules/Php74/NodeAnalyzer/ClosureArrowFunctionAnalyzer.php

```diff
@@ -84,7 +86,7 @@ function (Node $subNode) use ($referencedValues): bool {
                 }
 
                 foreach ($referencedValues as $referencedValue) {
-                    $isFoundInInnerUses = (bool) array_filter(
+                    $isFoundInInnerUses = $this->arrayChecker->isExists(
```

Comment:
This would be better naming IMO:

```suggestion
                    $isFoundInInnerUses = $this->arrayChecker->doesExist(
```

Reply:
updated :+1: https://github.com/rectorphp/rector-src/pull/3262/commits/7d9ed4f83f6e417448ad82a99d5f3e3d7e39aaad